tensorboard/components_polymer3/tf_dashboard_common/data-loader-behavior.ts

```diff
@@ -192,62 +192,51 @@ export function DataLoaderBehavior<Item, Data>(
           if (result.cancelled) {
             return;
           }
-          // Read-only property have a special setter.
           this.dataLoading = true;
-          // Promises return cacheKeys of the data that were fetched.
-          const promises = this.dataToLoad
-            .filter((datum) => {
-              const cacheKey = this.getDataLoadName(datum);
-              return !this._dataLoadState.has(cacheKey);
-            })
-            .map((datum) => {
-              const cacheKey = this.getDataLoadName(datum);
-              this._dataLoadState.set(cacheKey, LoadState.LOADING);
-              return this.requestData(datum).then(
-                this._canceller.cancellable((result) => {
-                  // It was resetted. Do not notify of the response.
-                  if (!result.cancelled) {
-                    this._dataLoadState.set(cacheKey, LoadState.LOADED);
-                    this.loadDataCallback(this, datum, result.value as any);
-                  }
-                  return cacheKey;
-                })
-              );
-            });
-          return Promise.all(promises)
-            .then(
-              this._canceller.cancellable((result) => {
-                // It was resetted. Do not notify of the data load.
-                if (!result.cancelled) {
-                  const keysFetched = result.value as any;
-                  const fetched = new Set(keysFetched);
-                  const shouldNotify = this.dataToLoad.some((datum) =>
-                    fetched.has(this.getDataLoadName(datum))
-                  );
-                  if (shouldNotify) {
-                    this.onLoadFinish();
-                  }
-                }
-                const isDataFetchPending = Array.from(
-                  this._dataLoadState.values()
-                ).some((loadState) => loadState === LoadState.LOADING);
-                if (!isDataFetchPending) {
-                  // Read-only property have a special setter.
-                  this.dataLoading = false;
-                }
-              }),
-              // TODO(stephanwlee): remove me when we can use
-              // Promise.prototype.finally instead
-              () => {}
-            )
-            .then(
-              this._canceller.cancellable(({cancelled}) => {
-                if (cancelled) {
-                  return;
+          const dirtyItems = this.dataToLoad.filter((datum) => {
```

Comment:
Sorry about pre-existing naming I created. Can we, in a follow up, rename these (or create an alias) to say `item` instead of `data`? i.e., we use both right now with `dataToLoad`, `getDataLoadName`, and `dataLoadState` but use `item` in requestData`.